### PR TITLE
Switch to literal string interpolation

### PR DESCRIPTION
**Pull request by bgurney-rh**
Supersedes: #885
Related: stratis-storage/project#456

### CODE REVIEW

File: src/stratis_cli/_actions/_formatting.py [hunk 1]
```diff
@@ -42,10 +42,10 @@ def size_triple(size, used):
     """
     free = None if size is None or used is None else size - used
 
-    return "%s / %s / %s" % (
-        TABLE_FAILURE_STRING if size is None else size,
-        TABLE_FAILURE_STRING if used is None else used,
-        TABLE_FAILURE_STRING if free is None else free,
+    return (
```
Comment (mulkieran) [inline src/stratis_cli/_actions/_formatting.py:45]: Copy-paste error here.

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: src/stratis_cli/_error_reporting.py [hunk 2]
```diff
@@ -310,17 +307,13 @@ def _interpret_errors_2(errors):
...
-                    return fmt_str % (
-                        context.property_name,
-                        error.interface_name,
-                        context.value,
-                        next_error.get_dbus_message(),
+                    return (
+                        f"stratisd failed to perform the operation that you "
+                        f"requested, because it could not set the D-Bus "
+                        f'property "{context.property_name}" belonging to '
+                        f'interface "error.interface_name" to "{context.value}". '
```
Comment (mulkieran) [inline src/stratis_cli/_error_reporting.py:314]: Needs curly braces!

**Review by mulkieran — CHANGES_REQUESTED** [review]
Caught just one thing...

File: src/stratis_cli/_error_reporting.py (hunk 2 above)
Comment (bgurney-rh) [reply src/stratis_cli/_error_reporting.py]: Resolved.

**Review by mulkieran — APPROVED** [review]

**Comment by mulkieran** [comment]
@bgurney-rh Plz rebase.

**Comment by bgurney-rh** [comment]
Rebased.

**Pull request merged**